subprojects/core/src/main/resources/META-INF/groovy/org.codehaus.groovy.runtime.ExtensionModule

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright 2022 the original author or authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+moduleName=dependency-management
+moduleVersion=1.0
+extensionClasses=org.gradle.api.internal.artifacts.dsl.dependencies.ConfigurationDependencyHandlerExtensionModule
```

Comment:
Could you rename this to match the `DependencyAdder` name?

Reply:
Done.